src/picomsg/src/config_conn/ConfigConn.cpp

```diff
@@ -177,8 +177,7 @@ class ConfigServiceImpl final : public config_conn_api::Config::Service {
 
 
 		std::lock_guard _guard(m_lock);
-
-		if (! m_pm_params.add_system_enterprise_cb(m_conn, request->terminator_id(), request->tenant_id().c_str(), request->account_id().c_str(), request->vrf_name().c_str(), term_vlan_id, &term_private_addr)) {
+		if (! m_pm_params.add_system_enterprise_cb(m_conn, request->terminator_id(), request->tenant_id().c_str(), request->account_id().c_str(), request->vrf_name().c_str(), term_vlan_id, &term_private_addr, request->neigh_sub_interface().c_str())) {
```

Comment:
Keep the empty line after locking `m_lock`. It is used to clearly mark the critical section